changelog.txt

```diff
@@ -2,14 +2,19 @@ V.Next
 ----------
 - [MAJOR] Adding YubiKit SDK, which requires Java Version 8 and will thus bump up Java version overall to 8; added keyboard flag to android:configChanges for all activities that could interact with a YubiKey. (#1729)
 - [MINOR] Support WordApp local apk install in UI automation flows (#1732)
-- [PATCH] Add exception handling in Content Provider strategy, for broker communication (#1722)
-- [MINOR] Support TenantID value from eSTS in PKeyAuth flows (#1712)
-- [MINOR] Implement cert loader for both multiple and legacy WPJ data store in PKeyAuth (#1711)
 - [PATCH] Fix SDK Cancel when using authorization in current task (#1743)
 - [MINOR] Added support for broadcasting to applications installed on the device. (#1744)
 - [PATCH] Fix APPLICATION_CANCELLED by handling back button press (#1725)
 - [PATCH] Added Base64 encode for the AuthorizationRequest state parameter (#1750)
 
+Version 5.0.0
```

Comment:
Can you also add an entry here for [4.2.0 release](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/release/4.2.0/changelog.txt), and then update the changes in 5.0.0

Version 4.2.0
----------
- [PATCH] Add exception handling in Content Provider strategy, for broker communication (#1722)
- [MINOR] Support TenantID value from eSTS in PKeyAuth flows (#1712)
- [MINOR] Implement cert loader for both multiple and legacy WPJ data store in PKeyAuth (#1711)